search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -144,10 +135,8 @@ class TemporalEntityAttributeService(
     fun getForEntity(id: URI, attrs: Set<String>): Flux<TemporalEntityAttribute> {
         val selectQuery =
             """
-            SELECT id, temporal_entity_attribute.entity_id, type, attribute_name, attribute_value_type,
-                payload::TEXT, dataset_id
-            FROM temporal_entity_attribute
-            LEFT JOIN entity_payload ON entity_payload.entity_id = temporal_entity_attribute.entity_id            
+            SELECT id, temporal_entity_attribute.entity_id, type, attribute_name, attribute_value_type, dataset_id
```

Comment:
you can now remove the `temporal_entity_attribute` prefix (no longer ambiguous since no more join on `entity_payload` table)